features/step_definitions/volunteer_op_steps.rb

```diff
@@ -22,4 +22,8 @@
   click_on 'Edit'
   fill_in('Description', :with => description)
   click_on 'Update a Volunteer Opportunity'
+end
+
+Given /^I should see (\d+) markers in the map$/ do |num|
```

Comment:
Ambiguous regexp literal. Parenthesize the method arguments if it's surely a regexp literal, or add a whitespace to the right of the `/` if it should be a division.